docs/README.md

```diff
@@ -0,0 +1,34 @@
+# DOS Wasm X
+Welcome to the online documentation for DOS Wasm X.
+# Getting Started
+You will need:
+1. A release of DOS Wasm X **or** a Linux machine to compile from source
+2. A web server **or** Docker to host DOS Wasm X
+3. A decent computer with a modern web browser
+4. Something to play, like a game or an app
+
+First, get yourself a copy of DOS Wasm X.
+# Getting DOS Wasm X
+You can [download a release](https://github.com/nbarkhina/DosWasmX/releases) or [compile the emulator from source](https://nbarkhina.github.io/doswasmx/compile-from-source).

```

Comment:
Releases are created manually. Usually what I do is take the contents of my release, make a ZIP file and a TAR.GZ file (which I typically automate with a PowerShell script because it's easier), calc the SHA256 (if you want to), then you start a Release from the project homepage on Github. You can attach files to a release and type in release notes. Releases remain in draft state until you publish them, but you can always delete or edit them later (or make a new release which you mark as Latest), so they're not permanent.

Reply:
Releases are really for ease of use, for people who just want your latest build without having to read through the repo details. You can also tie them to Issue tracking if you want, so you can link resolved issues to a release, but for users they're a simple easy way to grab your latest build.

Reply:
Sounds good, we may want to and a Changelog to the repo to track what went into each release.